### PR TITLE
Remove the hardcoded password for FSP

### PR DESCRIPTION
**Pull request by zet809**
For issue #1750

### CODE REVIEW

**Comment by zet809** [comment]
@whowutwut Since those 2 options 'dev' and 'celogin1' is never used, and it is them that used the hard coded passwords. I remove those 2 options with this pull request, pls help to review. Thx!

File: xCAT-server/lib/perl/xCAT/PPC.pm [hunk 1]
```diff
@@ -2220,7 +2220,6 @@ sub process_request {
 
         #For using rspconfig options through ASMI
         #my $arg = $request->{arg};
-        #if($request->{command} eq "rspconfig" and grep(/^(dev|celogin1)/, @$arg)) {
```
Comment (whowutwut) [inline xCAT-server/lib/perl/xCAT/PPC.pm]: While you have modified this code, can you also delete that comment line 2222?

Comment (zet809) [inline xCAT-server/lib/perl/xCAT/PPC.pm]: Done, pls review.

**Comment by whowutwut** [comment]
I have reviewed, will merge. Thanks!

**Pull request merged**